eve/workers/master/Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM ubuntu:trusty
+
+#
+# Install apt packages needed by the buildchain
+#
+ENV LANG C.UTF-8
+COPY buildbot_worker_packages.list vaultclient_packages.list /tmp/
+RUN apt-get update -q && apt-get -qy install curl apt-transport-https \
+    && apt-get install -qy software-properties-common python-software-properties \
+    && curl --silent https://deb.nodesource.com/gpgkey/nodesource.gpg.key | apt-key add - \
+    && echo "deb https://deb.nodesource.com/node_6.x trusty main" > /etc/apt/sources.list.d/nodesource.list \
+    && add-apt-repository ppa:ubuntu-toolchain-r/test \
+    && apt-get update -q \
+    && cat /tmp/buildbot_worker_packages.list | xargs apt-get install -qy \
+    && cat /tmp/vaultclient_packages.list | xargs apt-get install -qy \
+    && pip install pip==9.0.1 \
+    && rm -rf /var/lib/apt/lists/* \
+    && rm -f /tmp/*_packages.list
+
+#
+# Install usefull nodejs dependencies
+#
+
+RUN npm install mocha -g
+
+#
+# Add user eve
+#
+
+RUN adduser -u 1042 --home /home/eve --disabled-password --gecos "" eve \
+    && adduser eve sudo \
+    && sed -ri 's/(%sudo.*)ALL$/\1NOPASSWD:ALL/' /etc/sudoers
```

Comment:
Looks a bit too permissive to me. Can we easily set it up for the eve user only, may be ?

Reply:
We can try adding only `eve`.

Reply:
I will, but since eve is the only user on the sudo group it doesn't bring a real change